### PR TITLE
Adding lib angular-translate 

### PR DESCRIPTION
**Pull request by ArthurMota9**
# Summary

Angular-translate is an AngularJS module that makes your life much easier when it comes to **i18n**. It provides components like **filters** and **directives**, asynchronous loading of i18n data, and much more.

Two files were created with the English translations `app/i18n/languages/en.js` and Portugueses translations `app/i18n/languages/pt_BR.js.`, the files are JS objects with key and value, the same key must be inserted in both files, changing only the value.

At that moment, only a few words were inserted to test the functionality, in a second moment, all the words had to be inserted for complete usability.

[Angular-translate ocumentation here.](https://angular-translate.github.io/)

# List of changes

- [Adding angular-translate lib;](https://github.com/brmodeloweb/brmodelo-app/pull/152/commits/8a11bd64c416a9d1fee68a25ff104006eaa2dfe3)
- [Adding flag svgs and style;](https://github.com/brmodeloweb/brmodelo-app/pull/152/commits/34b3fe8f80d96e0d6285974aaefb9d1145d1e3f4)
- [Config angular-translate and add some words.
](https://github.com/brmodeloweb/brmodelo-app/pull/152/commits/d13ded2c26806eb2dcc5699fe756461f97044fa3)

# Records
https://user-images.githubusercontent.com/25749372/113516180-34757500-954f-11eb-8f63-2f2faf5eb14c.mov

### CODE REVIEW

File: app/sass/public.scss [hunk 1]
```diff
@@ -60,4 +60,25 @@
...
+// .public-footer
+////////////////////////////////////////////////////////////////////////////////
+
+.public-footer {
+	display: flex;
+	justify-content: center;
+	width: 100%;
+	margin-top: 40px;
+
+	.country-flag:first-of-type {
```
Comment (idmarjr) [inline app/sass/public.scss]: By default we dont use Sass _nesting_ in our `.scss`files.
Can you remove the nesting from here, please? :)

Comment (ArthurMota9) [reply app/sass/public.scss]: Done!

**Comment by ArthurMota9** [comment]
@miltonbsn and @oicramps, can you review, please?

File: app/angular/login/login.html [hunk 2]
```diff
@@ -16,38 +16,43 @@ <h1>BR Modelo</h1>
...
+						placeholder="{{ 'Email' | translate }}"
 						autocomplete="off"
 						required
 						ng-model="$ctrl.credentials.username"
 						data-ng-class="{'error': loginForm.email.$invalid && (!loginForm.email.$pristine || $ctrl.submitted)}"
 					/>
 				</div>
 				<div class="form-group">
-					<label for="exampleInputPassword1">Password</label>
+					<label for="exampleInputPassword1">{{ "Password" | translate }}</label>
```
Comment (oicramps) [inline app/angular/login/login.html]:
```suggestion
					<label for="exampleInputPassword1">{{ 'Password'| translate }}</label>
```

You're using single quotes in the other places, I think you can change here to keep a pattern

File: views/index.html [hunk 3]
```diff
@@ -53,6 +53,9 @@
 	<!-- Angular UI Bootsrtrap -->
 	<script src="angular-ui-bootstrap/dist/ui-bootstrap-tpls.js"></script>
 
+	<!-- Angular Translate -->
+	<script src="angular-translate/dist/angular-translate.min.js"></script>
```
Comment (oicramps) [inline views/index.html]: Your branch is based on the webpack migration branch, right? If yes, I think this script file isn't  necessary

**Review by oicramps — CHANGES_REQUESTED** [review]
Watching the example video seems that the Brazilian flag is bigger than the USA flag. Is that right?
image

File: app/angular/login/login.html (hunk 2 above)
Comment (ArthurMota9) [reply app/angular/login/login.html]: Done!

File: views/index.html (hunk 3 above)
Comment (ArthurMota9) [reply views/index.html]: I tried it here and worked without the script, thanks!

**Comment by ArthurMota9** [comment]
image

I updated the flags, I found some more beautiful!

**Comment by oicramps** [comment]
> image
> 
> I updated the flags, I found some more beautiful!

Much better now!

**Review by oicramps — APPROVED** [review]

**Pull request merged**